suite/tests/client-interface/drreg-test.c

```diff
@@ -255,6 +257,17 @@ handle_signal12(int signal, siginfo_t *siginfo, ucontext_t *ucxt)
     SIGLONGJMP(mark, 1);
 }
 
+static void
+handle_signal13(int signal, siginfo_t *siginfo, ucontext_t *ucxt)
```

Comment:
Should each numeric suffix on the handlers be replaced with the descriptive suffix for the corresponding test_ routine?

Reply:
Done. Some names could be a bit long because many signal handlers handle two tests.